ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -731,7 +731,7 @@ orderby worksheet.Position
                     var definedNameText = worksheet.PageSetup.PrintAreas.Aggregate(String.Empty,
                         (current, printArea) =>
                             current +
-                            ("'" + worksheetName + "'!" +
+                            (EscapeName(worksheetName) + "!" +
```

Comment:
The single quotes are necessary when the sheet name has a space too.

Reply:
That's why some tests are failing.

Reply:
I see. And that is why some of the existing tests fail.